broadcasting.md

```diff
@@ -247,6 +255,47 @@ npm run dev
 > **Note**  
 > To learn more about compiling your application's JavaScript assets, please consult the documentation on [Vite](/docs/{{version}}/vite).
 
+Additional supported features :
+
+*1. Modify private/presence channel capability. Default: Full capability*
+- Channel access control rights are granted for each individual user separately using `ably-capability`.
+- It defines list of access claims as per [Channel Capabilities](https://ably.com/docs/core-features/authentication#capability-operations).
```

Comment:
```suggestion
- Ably capability objects define a list of access claims as per [Channel Capabilities](https://ably.com/docs/core-features/authentication#capability-operations).
```

Reply:
Using objects term would be complex because it's always going to be a string array.

Reply:
I have put it as `Ably capability array define a list ...`

Reply:
makes sense, but you should change it to "Ably capability arrays..." because the singular is grammatically incorrect here

Reply:
Okay, I am not that good at grammar, hopefully it doesn't confuser users.

Reply:
done